src/index.js

```diff
@@ -32,7 +32,10 @@ async function createNewEventHandler(kc) {
       kubeClass: kc,
       logger: log,
       requestOptions: { qs: { timeoutSeconds: process.env.CRD_WATCH_TIMEOUT_SECONDS || 300 } },
-      livenessInterval: true
+      livenessInterval: true,
+      options: {
+        reconcileByDefault: process.env.MTP_RECONCILE_BY_DEFAULT || null
```

Comment:
curious your opinion.. would it make more sense to just specify `true` instead of `null`. Seems to me it would make the default behavior clearer from this file, but possibly more confusing if looking at the baseController code.

Reply:
Sure, seems sensible